src/cascadia/WindowsTerminal/IslandWindow.cpp

```diff
@@ -1905,5 +1905,26 @@ void IslandWindow::UseMica(const bool newValue, const double /*titlebarOpacity*/
     // that version, this API will just return an error and do nothing silently.
 
     const int attribute = newValue ? DWMSBT_MAINWINDOW : DWMSBT_NONE;
-    std::ignore = DwmSetWindowAttribute(GetHandle(), DWMWA_SYSTEMBACKDROP_TYPE, &attribute, sizeof(attribute));
+    LOG_IF_FAILED(DwmSetWindowAttribute(GetHandle(), DWMWA_SYSTEMBACKDROP_TYPE, &attribute, sizeof(attribute)));

```

Comment:
we're gonna log this on every windows 10 machine, and probably get smacked with a "garrulous event" bug

Reply:
This was marked resolved, but I don't understand how it's been resolved :smile:

Reply:
bahg saved but forgot to `git add`